### DISCUSSION TITLE
Side pads stopped working

### DISCUSSION THREAD

**Discussion by asdfjkluiop** [solokeys/solo2#166] [Q&A] [answered]
I just got my key today and was testing it out with some websites. After a few uses both of the side pads on the key stopped responding to touch however the back pad still works. I have a suspicion my key might have a hardware issue but figured I'd ask to see if anyone else has had a similar problem

**Comment by msiqx9** [comment, marked as answer]
That ia a hardware fault.  Get a warranty replacement.
